resources/assets/coffee/react/beatmap-discussions/main.coffee

```diff
@@ -136,7 +136,7 @@ class BeatmapDiscussions.Main extends React.PureComponent
       params.last_updated = moment(@state.beatmapsetDiscussion.updated_at).unix()
 
     if @state.beatmapsetDiscussion.beatmapset_events?
-      params.last_updated = _.max [params?.last_updated, moment(_.last(@state.beatmapsetDiscussion.beatmapset_events).created_at).unix()]
+      params.last_updated = _.max [params?.last_updated, moment(_.last(@state.beatmapsetDiscussion.beatmapset_events)?.created_at).unix()]
```

Comment:
wouldn't it just call `moment(undefined).unix()`? Maybe check `@state.beatmapsetDiscussion.beatmapset_events?[0]?` or something instead?